mxnet/training/docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -91,7 +91,7 @@ RUN pip install --no-cache --upgrade \
     # inotify-simple updated to 1.3.0 and has an issue that prevents the installation
     # of the enum34 package on py2. inotify-simple is used in sagemaker-mxnet-training
     "inotify-simple<1.3" \
-    "sagemaker-mxnet-training<4"
+    sagemaker-mxnet-training==4.2.0
```

Comment:
Could you set this to `"sagemaker-mxnet-training<5"`?

We need to be able to update the version of this package without having to manually change this version frequently.

Reply:
Addressed